src/web/components/MostViewedList/MostViewedList.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useApi } from '@root/src/web/components/lib/api';
+import { GuardianLines } from '@root/src/web/components/GuardianLines';
+import { css } from 'emotion';
+import { textSans } from '@guardian/src-foundations/typography';
+import { MostViewedListItem } from './MostViewedListItem';
+
+const wrapperStyles = css`
+    margin-top: 24px;
+`;
+
+const headingStyles = css`
+    ${textSans.medium({ fontWeight: 'bold' })}
+    margin-bottom: 8px;
+`;
+
+export interface TrailType {
+    url: string;
+    linkText: string;
+    isLiveBlog: boolean;
+    ageWarning: string;
+    image?: string;
+    pillar: Pillar;
+}
+
+export interface TabType {
+    heading: string;
+    trails: TrailType[];
+}
+
+interface Props {
+    limitItems?: number;
+}
+
+export const MostViewedList = ({ limitItems = 5 }: Props) => {
+    const endpointUrl: string =
+        'https://api.nextgen.guardianapps.co.uk/most-read-geo.json?dcr=true';
```

Comment:
To card up probably: I think we probably want to move this ajax domain to a config somewhere, maybe abstract to a getter - we have ajaxUrl in the CAPI response.

Reply:
Yeah that makes sense. I'm not sure this is the only place where we'd want to abstract the base URL. There might be others. But I agree we should at least move the base URL to some config object.

Card added to the backlog: https://trello.com/c/xv4b8EWl/868-abstract-baseurl-for-all-api-calls-into-separate-config-object